Makefile

```diff
@@ -36,3 +36,9 @@ fix:
 	bash ./scripts/fix.sh
 
 ci: mod lint tests
+
+api-generate:
+	goa gen github.com/keboola/keboola-as-code/design --output ./internal/pkg/template/api
```

Comment:
Generuju to rovnou do `internal/pkg/template/api`, protože v těch vygenerovaných `go` filech jsou importy a ty asi nechceme měnit nějakým find & replace (a navíc se to strká ještě do `gen` subfolderu s čímž nic neuděláme, ale klidně bych to v něm nechal ať je rovnou jasný že jsou to vygenerovaný soubory):
https://github.com/keboola/keboola-as-code/blob/a54bc3975eb3868321acbc7176fc434f657695a0/internal/pkg/template/api/gen/http/templates/server/encode_decode.go#L15

I když nad tím teď přemýšlím, že kdyby tam těch apiček bylo víc, tak tomu `goa gen` se nedá říct že chci generovat api jen pro ten jeden design (`api/templates.go`). Takže bysme to možná měli generovat do nějakýho top-level adresáře (ale tady jsem nenašel co by se k tomu hodilo: https://github.com/golang-standards/project-layout). Nebo alternativně to dávat do nějakých subfolderů (`design/templates/templates.go`). A nebo to můžeme vyřešit až se to stane a přibude nějaký nový apičko. 🙂

Reply:
Asi to sprav, ako je to podla teba nejlepsie, a pozriem to na review.
Ak to potrebujes prediskutovat hned, tak to mozem pozriet aj hned, ... len neviem ci mas v plane este nejake velke upravy tohto PR, alebo nie.

Reply:
Nene, jen jsem potřeboval fixnout ten linter a mám hotovo.

Reply:
ok, tak doriesim tie bugs a pozriem sa na to, ok?

Reply:
ok 👍  ještě dopíšu do docs jak se to ovládá